src/test/java/com/jcabi/s3/AwsBucketITCase.java

```diff
@@ -147,4 +150,17 @@ public void listsInPrefixedBucketWithouCollisions() throws Exception {
         }
     }
 
+    /**
+     * AwsBucket can check if the bucket exists.
+     * @throws IOException If fails
+     */
+    @Test
+    public void checkBucketExistence() throws IOException {
+        final Bucket bucket = this.rule.bucket();
+        Assert.assertTrue(bucket.exists());
```

Comment:
in this case, break this test method into two smaller ones. in general, always test one (!) thing/method in a single test method. mixing test targets together creates this type of confusion